packages/suite/src/components/settings/SettingsSection.tsx

```diff
@@ -1,24 +1,44 @@
 import React, { ReactNode } from 'react';
 
-import { Card, Column, IconName, InfoItem, useMediaQuery, variables } from '@trezor/components';
+import {
+    Card,
+    Column,
+    Icon,
+    IconName,
+    InfoItem,
+    Tooltip,
+    useMediaQuery,
+    variables,
+} from '@trezor/components';
 import { spacings } from '@trezor/theme';
 
 type SettingsSectionProps = {
     title: ReactNode;
     icon?: IconName;
     className?: string;
     children?: ReactNode;
+    tooltipText?: ReactNode;
 };
 
-export const SettingsSection = ({ title, icon, children }: SettingsSectionProps) => {
+export const SettingsSection = ({ title, icon, children, tooltipText }: SettingsSectionProps) => {
     const isBelowLaptop = useMediaQuery(`(max-width: ${variables.SCREEN_SIZE.LG})`);
 
     return (
         <InfoItem
+            allowTextWrap
             direction={isBelowLaptop ? 'column' : 'row'}
             labelWidth={250}
             iconName={icon}
-            label={title}
+            label={
+                <>
+                    {title}
+                    {tooltipText && (
+                        <Tooltip isInline content={tooltipText}>
```

Comment:
here we need somehow add top margin to the tooltip, so it sits nicely in the line, maybe line height hack could center it vertically to the line. @jvaclavik

Reply:
shouldn't we add `margin` from `frameProps` to `Tooltip`?

Reply:
as discussed: made with position: relative